keyboards/cornia/keymaps/fire/config.h

```diff
@@ -0,0 +1,33 @@
+/* Copyright 2024 Vaarai
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+/* Split */
+#define SPLIT_ACTIVITY_ENABLE
+#define SPLIT_LAYER_STATE_ENABLE
+#define SPLIT_LED_STATE_ENABLE
+#define SPLIT_MODS_ENABLE
+
+/* Trackpad */
+#define SPLIT_POINTING_ENABLE 
```

Comment:
Since this doesn't look to be reversible, this should probably be moved to te kb level, as well

Reply:
Pointing device is not seamlessly supported by the PCB, it require to solder extra things.

I prefer to keep this feature in the "advanced" keymap if it's ok for you ?